jni_gen/lib/src/util/find_package.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io';
+
+import 'package:package_config/package_config.dart';
+
+Future<Uri?> findPackage(String packageName) async {
+  final packageConfig = await findPackageConfig(Directory.current);
+  if (packageConfig == null) {
+    return null;
+  }
+
+  final packages = packageConfig.packages;
+  for (var candidate in packages) {
```

Comment:
Can you replace this loop with this function? https://pub.dev/documentation/package_config/latest/package_config.package_config_types/PackageConfig/operator_get.html